### ISSUE TITLE
Sample of creating an custom depth image and adding it to an image

### ISSUE THREAD

**Issue by dredenba** [shu223/iOS-Depth-Sampler#4]
Hi I was wondering if you could add a sample that could load up a grayscale jpg image and add that as the depth map of another image that does not already have depth data?

**Comment by mpc20001** [comment]
Yes I would also really really appreciate this as I have not been able to do it and have been working on it for 2 week.s
